### ISSUE TITLE
[Feature request] Add spinner to "DB backup/restore" windows

### ISSUE THREAD

**Issue by tarasyyyk** [TablePlus/TablePlus#1721]
1. Which TablePlus build number are you using: **296**

This is small UI improvement. It would be good to see spinner when databases loading by clicking on connections. On windows "Backup / Restore Database".
<img width="1367" alt="Restore Database 2020-02-12 22-02-49" src="https://user-images.githubusercontent.com/20100011/74373051-66c09200-4de4-11ea-81a2-4125cdd42694.png">

**Comment by ggedde** [comment]
Agreed, need some sort of indicator that the Backup or Restore process is still running.
I sometimes have GB databases being backed up or restored and can take several minutes.
It is hard to know if the process is still running when all I see is "Restoring...", but no visual indication.

Bonus Points if you can use a Progress bar so we can see how much longer it is going to take. Doesn't need to be 100% accurate, but a rough percentage bar would be ideal.

Thanks
